bpf/include/bpf/helpers.h

```diff
@@ -104,4 +104,6 @@ static int BPF_FUNC_REMAP(get_socket_opt, void *ctx, int level, int optname,
 			  void *optval, int optlen) =
 	(void *)BPF_FUNC_getsockopt;
 
+static __u64 BPF_FUNC(get_current_cgroup_id);
```

Comment:
This should be defined two commits earlier, when you start using it in `bpf/lib/trace_sock.h`, otherwise, the per-commit compilation may fail (it doesn't fail in tests because `TRACE_SOCK_NOTIFY` is not set)

Reply:
Good catch! Fixed. Thanks!